### DISCUSSION TITLE
bug: Format Document does nothing

### DISCUSSION THREAD

**Discussion by alsimoneau** [krvajal/vscode-fortran-support#775] [Q&A - deprecated see Fortran Discourse]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When I try to format a document using either formatter, nothing happens.

`fprettify` is installed and on my `$PATH`, I have also tried passing the path explicitly in multiple ways based on  #582 and #669.

Here is my minimal config file to reproduce:

```
{
    "fortran.formatting.formatter": "fprettify",
    "fortran.formatting.path": "/home/alsimoneau/.local/bin"
}
```

### Expected Behaviour

The code to be formatted

### Version of Modern Fortran

v3.2.0

### Version of Visual Studio Code

v1.74.0

### Platform and Architecture

Ubuntu

**Comment by gnikit** [comment]
Have you tried the pre-release version of the extension?

**Comment by alsimoneau** [comment]
It seems to work. Thanks.

However, fprettify does not seem to find the config file when called from VSCode. I don't know if it's something you have control over.

**Comment by gnikit** [comment]
Try specifying the absolute path to your configuration file, currently the extension does not resolve relative paths for the formatters' configuration options
